Exam1/Problem1/main.py

```diff
@@ -46,17 +45,14 @@ def GetMagnitude(x,y,x_2,y_2):
     y_help_list = []
     x_error_list =[]
     y_error_list = []
-    for dt in np.arange(0.0, max_time,t_setp):
+    for dt in np.arange(0.0, max_time,t_step):
         # for dt in range()
         
-        psi =  psi + Psi_dot *t_setp
```

Comment:
This was the problem that I found. This is computing the next step, but using the current dt as the parameter for GetExactCoord, so the discrete function was always one step ahead.

Reply:
So I moved it to the end of this for loop. If you want to see the difference, you can set t_list = [0.01] and max_time = 0.05. You will see that they start in the same place